fastlane/lanes/release.rb

```diff
@@ -0,0 +1,236 @@
+  lane :complete_code_freeze do |options|
+    ios_completecodefreeze_prechecks(options)
+    generate_strings_file_for_glotpress
+
+    UI.confirm('Ready to push changes to remote and trigger the beta build?') unless ENV['RELEASE_TOOLKIT_SKIP_PUSH_CONFIRM']
+    push_to_git_remote(tags: false)
+    trigger_beta_build(branch_to_build: "release/#{ios_get_app_version}")
+  end
+
+  # Creates a new beta by bumping the app version appropriately then triggering a beta build on CI
+  #
+  # @option [Boolean] skip_confirm (default: false) If true, avoids any interactive prompt
+  # @option [String] base_version (default: _current app version_) If set, bases the beta on the specified version
+  #                  and `release/<base_version>` branch instead of the current one. Useful for triggering betas on hotfixes for example.
+  #
+  desc 'Trigger a new beta build on CI'
+  lane :new_beta_release do |options|
+    ios_betabuild_prechecks(options)
+    download_localized_strings_and_metadata(options)
+    # FIXME: (2021.06.17) This is disabled because we currently have a >256 chars string which GlotPress truncates when exporting  the `.strings` files,
+    #   leading to incorrect key for it and (rightful) linter failure. We need to split that key into 2 smaller copies before we can re-enable this.
+    # ios_lint_localizations(input_dir: 'WordPress/Resources', allow_retry: true)
+    ios_bump_version_beta
+    version = ios_get_app_version
+    trigger_beta_build(branch_to_build: "release/#{version}")
+  end
+
+  # Sets the stage to start working on a hotfix
+  #
+  # - Cuts a new `release/x.y.z` branch from the tag from the latest (`x.y`) version
+  # - Bumps the app version numbers appropriately
+  #
+  # @option [Boolean] skip_confirm (default: false) If true, avoids any interactive prompt
+  # @option [String] version (required) The version number to use for the hotfix (`"x.y.z"`)
+  #
+  desc 'Creates a new hotfix branch for the given `version:x.y.z``. The branch will be cut from the tag `x.y` of the previous release'
+  lane :new_hotfix_release do |options|
+    prev_ver = ios_hotfix_prechecks(options)
+    ios_bump_version_hotfix(
+      previous_version: prev_ver,
+      version: options[:version],
+      skip_deliver: true
+    )
+  end
+
+  # Finalizes a hotfix, by triggering a release build on CI
+  #
+  # @option [Boolean] skip_confirm (default: false) If true, avoids any interactive prompt
+  #
+  desc 'Performs the final checks and triggers a release build for the hotfix in the current branch'
+  lane :finalize_hotfix_release do |options|
+    ios_finalize_prechecks(options)
+    version = ios_get_app_version
+    trigger_release_build(branch_to_build: "release/#{version}")
+  end
+
+  # Finalizes a release at the end of a sprint to submit to the AppStore
+  #
+  #  - updates store metadata
+  #  - bumps final version number
+  #  - removes branch protection and close milestone
+  #  - triggers the final release on CI
```

Comment:
For consistency with the formatting in the other lanes:

```suggestion
  #  - Updates store metadata
  #  - Bumps final version number
  #  - Removes branch protection and close milestone
  #  - Triggers the final release on CI
```